OctopusProjectBuilder.Uploader/Converters/TeamConverter.cs

```diff
@@ -15,9 +16,9 @@ public static async Task<TeamResource> UpdateWith(this TeamResource resource, Te
             resource.ExternalSecurityGroups = new NamedReferenceItemCollection();
             foreach (var esg in model.ExternalSecurityGroups)
                 resource.ExternalSecurityGroups.Add(new NamedReferenceItem { Id = esg });
-            resource.UserRoleIds = new ReferenceCollection(await Task.WhenAll(model.UserRoles.Select(async ur => await repository.UserRoles.ResolveResourceId(ur))));
-            resource.ProjectIds = new ReferenceCollection(await Task.WhenAll(model.Projects.Select(async p => await repository.Projects.ResolveResourceId(p))));
-            resource.EnvironmentIds = new ReferenceCollection(await Task.WhenAll(model.Environments.Select(async e => await repository.Environments.ResolveResourceId(e))));
+            //resource.UserRoleIds = new ReferenceCollection(await Task.WhenAll(model.UserRoles.Select(async ur => await repository.UserRoles.ResolveResourceId(ur))));
+            //resource.ProjectIds = new ReferenceCollection(await Task.WhenAll(model.Projects.Select(async p => await repository.Projects.ResolveResourceId(p))));
+            //resource.EnvironmentIds = new ReferenceCollection(await Task.WhenAll(model.Environments.Select(async e => await repository.Environments.ResolveResourceId(e))));
```

Comment:
What is the reason for commenting out this code and loosing the team mapping capability?